projects/1.18/assets/enlightened-end/nourished_end/lang/zh_cn.json

```
@@ -0,0 +1,292 @@
+  "block.nourished_end.verdant_eylium": "翠绿菌石",
+  "item.minecraft.lingering_potion.effect.illuminating": "滞留型发光药水",
+  "item.minecraft.potion.effect.vigorous_potion": "活力药水",
+  "item.nourished_end.end_rye": "末地黑麦",
+  "block.nourished_end.cracked_malachite_tiles": "裂纹孔雀石瓷砖",
+  "subtitles.ennegel_place": "方块：被放置",
+  "item.nourished_end.voidsteel_armor_chestplate": "精金胸甲",
+  "block.nourished_end.shulker_shell_block": "壳砖",
+  "block.nourished_end.tube_sprouts": "管状新芽",
+  "block.nourished_end.cataclysmic_sand": "毁灭尘碎石",
+  "block.nourished_end.verdant_pressure_plate": "翠绿压力板",
+  "item.nourished_end.ringling_spawn_egg": "林灵刷怪蛋",
+  "item.nourished_end.voidsteel_armor_helmet": "精金头盔",
+  "block.nourished_end.malachite_ore": "孔雀石矿石",
+  "block.nourished_end.malachite_tile_slab": "孔雀石瓷砖台阶",
+  "block.nourished_end.seldge_pressure_plate": "塞尔奇压力板",
+  "item.minecraft.tipped_arrow.effect.equalized_potion": "平衡之箭",
+  "advancements.bucket_manca.descr": "用碗拿起一只甲壳虫宝宝",
+  "block.nourished_end.dim_malachite_lamp": "孔雀石灯",
+  "subtitles.step_ender_slime": "脚印",
+  "block.nourished_end.brown_shell_tiles": "棕色壳砖",
+  "item.nourished_end.flarebug_spawn_egg": "萤火虫刷怪蛋",
+  "item.minecraft.tipped_arrow.effect.vigorous_potion": "活力之箭",
+  "block.nourished_end.seldge_button": "塞尔奇按钮",
+  "block.nourished_end.isopod_nest": "甲壳虫巢穴",
+  "item.nourished_end.flarebug_sac": "萤火虫囊",
+  "entity.nourished_end.giant_isopod": "甲壳虫",
+  "item.nourished_end.tall_music_disc.desc": "Naps the Block - Tall",
+  "item.nourished_end.isopod_spawn_egg": "甲壳虫宝宝刷怪蛋",
+  "item.nourished_end.voidsteel_shovel": "精金锹",
+  "block.nourished_end.verdant_shroomlight": "翠绿菌光体",
+  "block.nourished_end.malachite_lamp": "孔雀石灯笼",
+  "block.nourished_end.seldge_fan_plant_empty": "塞尔奇环形蕨茎",
+  "block.nourished_end.malachite_wall": "孔雀石墙",
+  "item.minecraft.potion.effect.equalized_potion": "平衡药水",
+  "advancements.radiation_fever.title": "辐射热",
+  "block.nourished_end.cataclysmic_sand_falling": "毁灭尘沙",
+  "block.nourished_end.malachite_brick_wall": "孔雀石砖墙",
+  "block.nourished_end.watcher_off": "观察者",
+  "block.nourished_end.seldge_fence_gate": "塞尔奇栅栏门",
+  "block.nourished_end.polished_nullstone_wall": "磨制虚空石墙",
+  "gamerule.visopodNestSpawning": "甲壳虫巢穴生成",
+  "block.nourished_end.cerulean_button": "晶蓝按钮",
+  "block.nourished_end.nullstone_outcrop": "虚空石尖角",
+  "entity.nourished_end.isopod": "甲壳虫宝宝",
+  "gamerule.malachiteToxcitity": "孔雀石毒性",
+  "block.nourished_end.gray_shell_tiles": "灰色壳砖",
+  "item.nourished_end.cooked_stalker": "熟潜行者肉",
+  "block.nourished_end.raw_voidsteel_block": " 精金碎块方块",
+  "block.nourished_end.verdant_fence": "翠绿栅栏",
+  "block.nourished_end.cerulean_underlily": "晶蓝吊兰",
+  "block.nourished_end.tube_sprouts_empty": "翠绿管状植物",
+  "block.nourished_end.malachite_tile_stairs": "孔雀石瓷砖楼梯",
+  "block.nourished_end.endrye_1": "末地黑麦种子",
+  "block.nourished_end.cerulean_trapdoor": "晶蓝活板门",
+  "gamerule.visopodNestSpawning.description": "如果设置为 （false）,甲壳虫宝宝将不会从甲壳虫巢穴中生成",

```

```suggestion
  "gamerule.visopodNestSpawning.description": "如果设置为false，甲壳虫宝宝将不会从甲壳虫巢穴中生成",
```
此外，`甲壳虫宝宝`怎么听着有点怪......